PROTOCOL.md

```diff
@@ -431,6 +334,84 @@ Field    | Type | Description
 `errorMessage` | `string` | Human-readable message describing the error.
 `errorCode` | `string` | Machine-readable string describing the type of error.
 
+### Messages between tracker and node
+
+These messages are exchanged between trackers and nodes for peer discovery and topology formation.
+
+### StatusMessage
+
+A node reports its status to the tracker using this message.
+
+```
+[version, type, status]
+```
+Example:
+```
+[2, 14, {
+    streams: [],
+    started: 0,
+    rtts: {}
+}]
+```
+
+Field    | Type | Description
+-------- | ---- | --------
+`status` | `object` | state of node
+
+### InstructionMessage
+
+The tracker instructs nodes to connect to (and disconnect from) other nodes using this message.
+
+```
+[version, type, requestId, streamId, streamPartition, nodeAddresses, counter]
+```
+Example:
+```
+[2, 15, "request-id", "stream-id", ["ws://address1", "ws://address2"], 10]
+```
+
+Field    | Type | Description
+-------- | ---- | --------
+`streamId` | `string` | stream id
+`streamPartition` | `number` | stream partition
+`nodeAddresses` | `array` | list of nodes the receiving node should be connected to
+`counter` | `number` | incrementing counter value to keep track of latest instruction
+
+### FindStorageNodesMessage
```

Comment:
Recommend renaming the messages to `StorageNodesRequest` and `StorageNodesResponse` to conform to protocol convention and make it clearer which messages are related

Reply:
Agree. Changing.